examples/with-docker-compose/README.md

```diff
@@ -0,0 +1,70 @@
+# With Docker Compose
+
+This example contains everything needed to get a Next.js development and production environment up and running with Docker Compose.
+
+## Benfits of Docker Compose
+
+- Develop locally without Node.js or TypeScript installed ✨
+- Easy to run, consistent development environment across Windows, MacOS, and Linux teams
+- Run multiple Next.js apps, databases, and other microservices in a single deployment
+- Multistage builds combined with [Output Standalone](https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files-experimental) outputs up to [85% smaller apps](#production) (Approximately 110 MB compared to 1 GB with create-next-app)
+- BuildKit engine builds multiple Docker images in parallel
+- Easy configuration with YAML files
+
+## How to use
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
+
+```bash
+npx create-next-app --example with-docker-compose with-docker-compose-app
+# or
+yarn create next-app --example with-docker-compose with-docker-compose-app
+```
+
+## Prerequisites
+
+- On Windows and MacOS, install [Docker Desktop](https://docs.docker.com/get-docker). Docker Desktop includes Docker Compose as part of the installation.
+- On Linux, run:
+
+```bash
+# Install Docker and Docker Compose for Linux
+wget -O - https://gist.githubusercontent.com/wdullaer/f1af16bd7e970389bad3/raw/install.sh | bash
+```
+
+## Development
+
+1. Run the development server:
+
+```bash
+./dev.sh
```

Comment:
Where do I run this from? The root? What is this magic script? 😄

Reply:
Good point. I simplified and moved the contents of this magic script to the README.